pkg/ddl/executor.go

```diff
@@ -10139,19 +10154,20 @@ func (d *ddl) addBatchDDLJobsV1(jobWs []*JobWrapper) {
 		logutil.DDLLogger().Info("add DDL jobs",
 			zap.Int("batch count", len(jobWs)),
 			zap.String("jobs", jobs),
-			zap.Bool("table", toTable))
+			zap.Bool("table", toTable),
+			zap.Bool("fast_create", fastCreate))
 	}
 }
 
 // addBatchLocalDDLJobs gets global job IDs and delivery the DDL jobs to local TiDB
 func (d *ddl) addBatchLocalDDLJobs(jobWs []*JobWrapper) {
```

Comment:
will not be called currently?

Reply:
yes, some code are not removed currently, as this pr is quite large, will do it later